wordle-web/app.vue

```diff
@@ -5,6 +5,8 @@
         <template v-slot:prepend>
           <v-icon color="secondary"> mdi-book </v-icon>
         </template>
+        <v-btn @click="showUserNameDialog = true">{{ displayedUserName }}</v-btn>
```

Comment:
I dont see the username displayed? I could be wrong

Reply:
Its in the app bar, I though it was more pleasing to look at.